kitchen.yml

```diff
@@ -19,6 +19,9 @@ platforms:
         - echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen
         - locale-gen en_US.UTF-8
       platform: debian
+    provisioner:
```

Comment:
As you're repeating the same provisioner options for all suites, you can factorize them under L64

Reply:
Actually I'd prefer to let it like that. The plan is to add tests on other OSes later.